src/harness/projectsRunner.ts

```diff
@@ -428,16 +422,16 @@ class ProjectRunner extends RunnerBase {
         }
 
         function getErrorsBaseline(compilerResult: CompileProjectFilesResult) {
-            const inputFiles = compilerResult.program ? ts.map(ts.filter(compilerResult.program.getSourceFiles(),
-                sourceFile => !Harness.isDefaultLibraryFile(sourceFile.fileName)),
-                sourceFile => {
-                    return {
-                        unitName: ts.isRootedDiskPath(sourceFile.fileName) ?
-                            RunnerBase.removeFullPaths(sourceFile.fileName) :
-                            sourceFile.fileName,
-                        content: sourceFile.text
-                    };
-                }) : [];
+            const inputFiles = ts.map(compilerResult.configFileSourceFiles.concat(

```

Comment:
Nit: break up into multiple lines, filter inside ternary inside concat inside map is hard to read.